packages/client/src/utils.ts

```diff
@@ -29,10 +30,17 @@ export function setEthProvider (provider: any): any {
   return ethProvider
 }
 
+export function getAuroraProvider (): any {
+  auroraJsonRpcProvider = () => new ethers.providers.JsonRpcProvider({
```

Comment:
We are removing process.env from the library. `getAuroraProvider` should return the provider which was set by `setAuroraProvider`.